vulcan/lib/server/workflows/scripts/magma_retrieve_paths.py

```diff
@@ -1,47 +1,47 @@
 from archimedes.functions.dataflow import output_tsv, input_var
-from archimedes.functions.metis import metis_file_path
-from archimeded.functions.magby import Magby
+from archimedes.functions.magby import Magby
 
-input_records = input_var('record_ids')
+input_records = input_var('record_ids').split(", ")
+token = input_var('token')
 
 # Improve these with magby.getModels or similar in future
 seq_pool_model_name = 'sc_rna_seq_pool' 
 seq_model_name = 'sc_rna_seq'
-h5_attr_name = 'raw_counts_h5'
+h5_attr_name = ['raw_counts_h5']
 project_name = "mvir1" # <-- should be some sort of environment variable in future
-magma = Magby.Magby('https://magma.ucsf.edu', get_token()) # <-- <-- same as above comment
+magma = Magby.Magby(url='https://magma.ucsf.edu', token=token) # <-- <-- same as above comment
 
 singlular_record_ids = []
 # Loop per record
-for id in input_records:
+for rec in input_records:
     
     # if a POOL, retrieve and add individual records
-    if id.find('POOL') != -1:
+    if rec.find('POOL') != -1:
         ids_ret = magma.retrieve(
-            projectName: project_name,
-            modelName: seq_pool_model_name,
-            recordNames: id,
-            attributeNames: seq_model_name,
-            dataType: str='meta')
+            projectName= project_name,
+            modelName= seq_pool_model_name,
+            recordNames= '[' + rec + ']',
```

Comment:
This is fixed in the last update, so you won't need to encapsulate a single string into a list, it will recognize data type internally.

Reply:
actually not even sure how line 23 would work - you should pass a string or a list not a string with `'[]'`

Reply:
Oh this section doesn't work yet... I was planning to ask if we could work on it together via zoom at some point.

But 👍 your first comment addresses something that I was also planning to report as a bug!

Reply:
Got it working!  Fixed in the latest commit.